packages/legacy/core/App/screens/Settings.tsx

```diff
@@ -224,8 +224,10 @@ const Settings: React.FC<SettingsProps> = ({ navigation }) => {
 
   const SectionHeader: React.FC<{ icon: string; title: string }> = ({ icon, title }) => (
     <View style={[styles.section, styles.sectionHeader]}>
-      <Icon name={icon} size={24} style={{ marginRight: 10, color: SettingsTheme.iconColor }} />
-      <Text style={[TextTheme.headingThree, { flexShrink: 1 }]}>{title}</Text>
+      <Icon accessible={false} name={icon} size={24} style={{ marginRight: 10, color: SettingsTheme.iconColor }} />
+      <Text accessibilityRole={'header'} style={[TextTheme.headingThree, { flexShrink: 1 }]}>
```

Comment:
This might be pedantic but I think these elements are headings, not headers

Reply:
I think that's a more accurate word but I don't have "heading" as an accessibility role and the description seems to fit as a "content section".

Docs: **header** Used when an element acts as a header for a content section (e.g. the title of a navigation bar).